src/main/kotlin/uk/gov/dluhc/printapi/database/repository/TemporaryCertificateRepository.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.dluhc.printapi.database.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.dluhc.printapi.database.entity.SourceType
+import uk.gov.dluhc.printapi.database.entity.TemporaryCertificate
+import java.util.UUID
+
+@Repository
+interface TemporaryCertificateRepository : JpaRepository<TemporaryCertificate, UUID> {
+
+    fun findByGssCodeInAndSourceTypeAndSourceReference(gssCodes: List<String>, sourceType: SourceType, sourceReference: String): TemporaryCertificate?
```

Comment:
Hi @NathanRussellValtech,

One more for the follow-up PR hopefully. I assume we will expect this method to return multiple temporary certificates, since we can have multiple temporary certificates for a given `gssCode`, `sourceType` and `sourceReference`.